book/src/themes.md

```diff
@@ -104,6 +104,7 @@ We use a similar set of scopes as
 [TextMate](https://macromates.com/manual/en/language_grammars) scopes.
 
 - `type` - Types
+- `constructor`
```

Comment:
Should move this after the next line, or `builtin` gets nested under `constructor` instead of `type`.

`constructor` also isn't a scope per https://www.sublimetext.com/docs/scope_naming.html so I've been thinking if there's a better scope for it maybe?

Reply:
There's `entity.name.function.constructor`; also should we add the other `entity.` scopes like `entity.name.struct` and `entity.name.enum` ? It would allow more precision if required.

Reply:
I've been avoiding meta:

> Meta scopes are used to scope larger sections of code or markup, generally containing multiple, more specific scopes. These are not intended to be styled by a color scheme, but used by preferences and plugins.

We can keep a toplevel scope for now.